README.md

```diff
@@ -2,109 +2,116 @@
  <img src="https://docs.freetubeapp.io/images/logoColor.png" width=500 align="center">
 </p>
 
-FreeTube is an open source desktop YouTube player built with privacy in mind.
-Use YouTube without advertisements and prevent Google from tracking you with their cookies and JavaScript.
-Available for Windows, Mac & Linux thanks to Electron.
+**Copylefted libre software desktop YouTube player** (with privacy).
```

Comment:
The previous line is better in my opinion.
`FreeTube is an open source desktop YouTube player built with privacy in mind.
Use YouTube without advertisements and prevent Google from tracking you with their cookies and JavaScript.`

Reply:
https://github.com/FreeTubeApp/FreeTube/pull/175/files is the software side of things.
"open source" isn't the OSI-trademarked "Open Source", and doesn't describe reality (by design) in having that problem in the first place. "Copyleft" describes something in addition to the base claim.
"googlevideo.com" is not blocked, so statements about no tracking are a bit dubious. Not that I really fixed that.

Reply:
Yeah, I do agree with the tracking part. However, in the clause it does clarify that it is for cookies and javascript, which to my knowledge, freetube does not store. Also the (with privacy) seems like a cherry on the top when it is one of the foundations for the project.

Reply:
You can't make an all-encompassing claim and then boil it down in down to inconsistent qualifications.

I don't know what is better, because it just seems like self-referencing the name just underneath the big "FreeTube" banner, in the project page called "FreeTube".
I am not looking for something that blocks cookies and Javascript, I am looking to use something that gives a good experience.

Nothing ever sold itself on the basis of security, and you are making no coherent claim for it.
"In mind" is weaker. "prevent Google from tracking you" is total, and the user is left to guess whether "cookies" and "JavaScript" is a credible or meaningful claim. Without other routing, one of the "main goals" seems like an adaptation as far as its validity goes. To tie that to its entire use is a mistake.

The website is even worse saying: " You can enjoy your favorite content and creators without your habits being tracked. All of your user data is stored locally and never sent or published to the internet."

Small "i" Internet. At best the user could grasp that whoever wrote it doesn't represent an advertisement behemoth…